Tests/Lua/lua-5.1.5/test/README

```diff
@@ -5,7 +5,7 @@ to be read, to see how Lua programs look.
 Here is a one-line summary of each program:
 
    bisect.lua		bisection method for solving non-linear equations
-   cf.lua		temperature conversion table (celsius to farenheit)
+   cf.lua		temperature conversion table (celsius to fahrenheit)
```

Comment:
I believe Fahrenheit should start with a capital letter because it's a name :) There are more similar comments just to make it simpler to fix it.